src/main/java/marquez/api/models/DatasetsResponse.java

```diff
@@ -14,14 +14,17 @@
 
 package marquez.api.models;
 
-import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonCreator;
 import java.util.List;
 import lombok.NonNull;
 import lombok.Value;
 
 @Value
 public class DatasetsResponse {
-  @NonNull
-  @JsonProperty("datasets")
-  List<DatasetResponse> responses;
+  @NonNull List<DatasetResponse> datasets;
+
+  @JsonCreator
```

Comment:
Do we need `@JsonCreator` if we are only serializing the response?

Reply:
I use it to deserialize the response in the integration test.

Reply:
Ahh, 👍